src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -30,12 +30,32 @@ import { EntityDiagram } from '../core';
 import { ComputationRoute } from './ComputationRoute';
 import { DefaultVariableRedirect } from './DefaultVariableRedirect';
 import Subsetting from './Subsetting';
-import { RecordController } from '@veupathdb/wdk-client/lib/Controllers';
+import {
+  ErrorBoundary,
+  RecordController,
+} from '@veupathdb/wdk-client/lib/Controllers';
 import GlobalFiltersDialog from '../core/components/GlobalFiltersDialog';
 import { Loading } from '@veupathdb/wdk-client/lib/Components';
 import ShowHideVariableContextProvider from '../core/utils/show-hide-variable-context';
 import NotesTab from './NotesTab';
 import ShareFromAnalysis from './sharing/ShareFromAnalysis';
+import { Alert } from '@material-ui/lab';
+
+const AnalysisTabErrorBoundary = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => (
+  <ErrorBoundary
+    children={children}
+    renderError={() => (
+      <Alert severity="error" style={{ marginTop: 10 }}>
+        We're sorry, something went wrong with this tab. Please change to
+        another tab or refresh to try again.
```

Comment:
I'd offer a link to let the user reload the tab, similar to [what our (in?)famous "oops" modal offers](https://github.com/VEuPathDB/EbrcWebsiteCommon/blob/bc5dd3077310a46457e95d85f39f341114d644a6/Client/src/component-wrappers/Error.tsx#L39).

```suggestion
        We're sorry, something went wrong with this tab. Please change to
        another tab or {<a href="" title="Reload the current tab.">reload this tab</a>} to try again.
```